exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/types/BinaryColumnTypeTests.kt

```diff
@@ -0,0 +1,41 @@
+package org.jetbrains.exposed.sql.tests.shared.types
+
+import org.jetbrains.exposed.dao.id.IntIdTable
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.binaryLiteral
+import org.jetbrains.exposed.sql.insertAndGetId
+import org.jetbrains.exposed.sql.selectAll
+import org.jetbrains.exposed.sql.tests.DatabaseTestsBase
+import org.jetbrains.exposed.sql.tests.shared.assertEqualLists
+import org.junit.Test
+
+class BinaryColumnTypeTests : DatabaseTestsBase() {
+    @Test
+    fun testBinaryColumn() {
+        val defaultValue = "default-value".toByteArray()
+        val tester = object : IntIdTable("testBinaryColumn") {
+            val bin = binary("bin", 256).default(defaultValue)
+        }
```

Comment:
Please also move over existing tests from `DDLTests.kt`: `testBinaryWithoutLength()` and `testBinary()`

Reply:
TODO fix with [EXPOSED-433](https://youtrack.jetbrains.com/issue/EXPOSED-433) Check all the literals for default/insert/read operations